client/app/bundles/course/assessment/submission/components/answers/TextResponse.jsx

```diff
@@ -82,10 +175,14 @@ function TextResponse({ question, readOnly, answerId, graderView }) {
 
   const editableAnswer = question.autogradable ? plaintextAnswer : richtextAnswer;
 
+  const solutionsTable = question.comprehension ?
```

Comment:
Can we avoid rendering in the case where `graderView` is `false`? Perhaps make `solutionsTable` a function?

Reply:
I've changed it into a function.